lib/Field/FieldContent.js

```diff
@@ -33,10 +48,12 @@ export function FieldContent({ children, className, instructions }) {
 FieldContent.propTypes = {
   children: node.isRequired,
   className: string,
-  instructions: element
+  instructions: element,
+  state: oneOf([ACTIVE, ADDED, DELETED, MOVED_UP, MOVED_DOWN, UNCHANGED])
```

Comment:
🚩Here's another annoying naming nag. I think state isnt a great name just because state can mean something else thats pretty core to react.

Status instead?

Reply:
Thanks, this is good feedback, `state` has now been renamed to `status` 👍